README.md

```diff
@@ -56,6 +56,24 @@ The part up to the semicolon was there before, only the latter part was appended
 Note: You should make sure that the user executing this (probably `www-data` or something similar) has the rights
 (1.) to execute the script and (2.) to write to the indicated log file.
 
+## How to run as docker container
+
+To run the script as a container, you need to build the image first.
```

Comment:
Please add a link to the Docker documentation on how to install it. No need for a lengthy description here, just a link would be fine.

Reply:
Sorry, didn't get to it.